eng/pipelines/runtime.yml

```diff
@@ -610,47 +610,6 @@ jobs:
     jobParameters:
       testGroup: innerloop
       liveLibrariesBuildConfig: ${{ variables.debugOnPrReleaseOnRolling }}
```

Comment:
I think we can incorporate the mono build-test-job in here too: https://github.com/dotnet/runtime/blob/5335f94a9747a64bd300ce06a4ce36a60f4ea71f/eng/pipelines/runtime.yml#L663

Although I can do that as a separate PR later if you like.

Reply:
I would suggest adding this in a follow up pr